src/test/components/ButtonWithPanel.test.js

```diff
@@ -4,27 +4,31 @@
 
 // @flow
 import * as React from 'react';
+import { render, fireEvent, cleanup } from 'react-testing-library';
 import ButtonWithPanel from '../../components/shared/ButtonWithPanel';
 import ArrowPanel from '../../components/shared/ArrowPanel';
-import renderer from 'react-test-renderer';
 
-describe('shared/ButtonWithPanel', function() {
-  it('renders a closed panel and opens it when asked', () => {
-    const button = renderer.create(
+describe('shared/ButtonWithPanel', () => {
+  afterEach(cleanup);
+
+  it('renders the ButtonWithPanel with a closed panel', () => {
+    const { container } = render(
       <ButtonWithPanel
         className="button"
         label="My Button"
+        open={false}
```

Comment:
nit: I think you can just remove the attribute as it's optional, just like it's used in https://github.com/devtools-html/perf.html/blob/670b68e3d78fc82a4f7913009b7f5e74bc0dc44b/src/components/app/MenuButtons.js#L100-L104